fdb-relational-core/src/test/java/com/apple/foundationdb/relational/recordlayer/metadata/SchemaTemplateSerDeTests.java

```diff
@@ -433,6 +433,59 @@ public void onlyQueriedSqlFunctionsAreCompiled() throws RelationalException {
         Assertions.assertTrue(peekingDeserializer.hasOneCompilationRequestFor("SqlFunction3"));
     }
 
+    @Test
+    public void schemaTemplateToBuilderPreservesIntermingledTablesFlag() {
```

Comment:
Given that I believe this was raised in response to #3480, would it be good to have a more explicit test of adding a temporary function to a schema template with intermingled tables?